models/base/snowplow_base_events.sql

```diff
@@ -1,2 +1,47 @@
+with source as (
 
-select * from {{ var('snowplow:events') }}
+    select * from
+
+    {% if var('snowplow:use_fivetran_interface') %}
```

Comment:
I'm a little concerned about this -- not sure how it works currently, but i recall in the past, you couldn't put `ref` statements inside of `if/else` blocks.

As long as you've tested this, I feel A-ok about it.